README.md

```diff
@@ -1,6 +1,6 @@
-# ASP.NET Core Feature Flags
+# .NET Feature Management

```

Comment:
This document is getting very long (but very useful). Do you think if it will be helpful if we can add an index section somewhere?

Reply:
Yeah a top level section for index sounds like a good idea.

Here's what I'm thinking, let me know if you think anything else should be in it.

## Index
* Feature Flags
  * Feature Flag Declaration
* Dynamic Features
  * Dynamic Feature Declaration
* ASP.NET Core Integration
* Targeting
* Custom Feature Providers
* Caching

Reply:
That list doesn't reflect the ordering we currently have in the doc, so would need to move things in the doc around a bit. For example, the ASP.NET Core integration section is baked in with feature flags right now and comes before dynamic features are explained

Reply:
The list looks good to me.

Reply:
I modified it a bit, but I think it looks good.